### ISSUE TITLE
Un Nest Menu Items to go to the Front and backend of the site. (Replace with Icons)

### ISSUE THREAD

**Issue by Flipback8** [soflyy/oxygen-bugs-and-features#2634]
**Describe the feature you'd like to see included in Oxygen.**
*A clear, detailed description of the feature you're requesting.*

I would prefer if the menu items to go to the front and backend where not nested.
![Oxygen Menu](https://snipboard.io/KrjeBV.jpg)
I would like these to be two Icons.

**What are the use cases for this feature?**
Get to front or backend with one click.

**Examples of this feature or functionality.**

Swiss Knife.
![Swiss Knife](https://snipboard.io/DtMG3p.jpg)

Bricks Builder
![Bricks Builder](https://snipboard.io/fPR9DQ.jpg)

**Comment by Jamie-owens** [comment]
@Flipback8 But this would be more clutter for the interface. Elijah mentioned keeping the interface clean so I doubt this will be considered.

**Comment by Flipback8** [comment]
2 functional icons that are constantly used don't make the interface cluttered.

**Comment by avocadesign** [comment]
I agree - Also I actually miss the text "Back to WP" - That was super helpful for clients - the icons are less discoverable in my opinion.

**Comment by Flipback8** [comment]
"Back to WP" on hover over an icon would work but writing the actual words on the bar would take up too much space. That would definitely cause too much clutter.

I keep clients out of Oxygen.
